modules/xmpp/JingleSessionPC.js

```diff
@@ -542,7 +542,11 @@ export default class JingleSessionPC extends JingleSession {
                         + 'source[xmlns="urn:xmpp:jingle:apps:rtp:ssma:0"]');
 
             ssrcs.each((i2, ssrcElement) => {
-                const ssrc = ssrcElement.getAttribute('ssrc');
+                const ssrc = Number(ssrcElement.getAttribute('ssrc'));
+
+                if (isNaN(ssrc)) {
+                    return;
```

Comment:
maybe a warning? If we get something not a number here it would mean there is a problem in Jicofo, right?